support/builders/tow-boot/default.nix

```diff
@@ -99,6 +99,9 @@ let
 
       # Intrusive opinionated patches
       ./patches/0001-Tow-Boot-sunxi-ignore-mmc_auto-force-SD-then-eMMC.patch
+
+      # ODROID N2 SPI support
+      ./patches/0001-Enable-the-SPI-on-the-ODROID-N2-by-default.patch
```

Comment:
Board-specific patches live in the board vendor's directory, and are included only for the given board.

Additionally we might want to include this patch only when relevant, as discussed.

Reply:
Isn't it always relevant in the u-boot stage though? Wouldn't it be convenient always to have access to the SPI in the boot stage? Not like you do any intensive computations in Tow-Boot, so I don't think the half eMMC bandwidth is that relevant.

Reply:
Yeah, "might" want. Not sure. You're probably right.